.github/workflows/build_and_test_clang_debug.yml

```diff
@@ -55,16 +57,36 @@ jobs:
                               -DKEEP_APOLLO_LOGS=TRUE\
                               -DUSE_FAKE_CLOCK_IN_TIME_SERVICE=TRUE\" "\
               && script -q -e -c "make test"
+        - name: Check ERROR/FATAL logs
+          id: err_report
```

Comment:
If you are using this step id  anywhere then you can remove it.

Reply:
right, removed it